piksi_tools/console/settings.yaml

```diff
@@ -404,6 +404,21 @@
     for each timestamp. When "No DGNSS" is chosen, no differential output will be
     attempted by the receiver.
 
+- group: solution
+  name: process_noise_motion_mode
+  expert: false
+  type: enum
+  units: N/A
+  default value: 'Default'
+  readonly: false
+  enumerated possible values: Default, Automotive, Static
+  Description: Selects the process noise variance of position, velocity & acceleration in the Horizontal &
```

Comment:
We probably also want something a little more customer friendly for the description